Added a class for a new feature to fix table header on top (#29456)
twbs/bootstrap: site/content/docs/4.3/content/tables.md

</table>
{{< /example >}}

### Sticky header

> Add `.thead-sticky` to stick it to the top of the table's parent container. Doing so only the body of the table will scroll in case it exceeds its height.

Reviewer: ```suggestion
Add `.thead-sticky` to a `<thead>` to make it stick to the top of the table's parent container while the `<tbody>` contents continue to scroll.
```
